docs/versioned_docs/version-2.x/components.mdx

```diff
@@ -209,13 +213,18 @@ word vectors in your pipeline.
       # Name of the language model to use
       model_name: "bert"
       # Pre-Trained weights to be loaded
-      model_weights: "bert-base-uncased"
+      model_weights: "rasa/LaBSE"
 
       # An optional path to a specific directory to download and cache the pre-trained model weights.
       # The `default` cache_dir is the same as https://huggingface.co/transformers/serialization.html#cache-directory .
       cache_dir: null
   ```
 
+  :::caution Changed in 2.1
+  Pre-trained weights for BERT have been renamed from `bert-base-uncased` to `rasa/LaBSE`.
```

Comment:
```suggestion
  The default pre-trained weights for BERT have been renamed from `bert-base-uncased` to `rasa/LaBSE`.
```

Reply:
added that this is the default because you can still use `bert-base-uncased`

Reply:
this is the last thing then I think we can merge

Reply:
awesome 👍🏻